Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -653,6 +654,7 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
 
     // MARK: - Tunnel Configuration
 
+    @MainActor
```

Comment:
Just want to make sure we're not updating the tunnel configuration multiple times in parallel.